core/arch/arm/arm.mk

```diff
@@ -96,6 +96,13 @@ endif
 # changes
 ifeq ($(CFG_CORE_SEL1_SPMC),y)
 $(call force,CFG_CORE_FFA,y)
+$(call force,CFG_CORE_SEL2_SPMC,n)
+endif
+# SPMC configuration "S-EL2 SPMC" where SPM Core is implemented at S-EL2,
+# that is, the hypervisor sandboxing OP-TEE
+ifeq ($(CFG_CORE_SEL2_SPMC),y)
+$(call force,CFG_CORE_FFA,y)
+$(call force,CFG_CORE_SEL1_SPMC,n)
```

Comment:
We could delete this line. If CFG_CORE_SEL1_SPMC=y then CFG_CORE_SEL2_SPMC will be set to n

Reply:
OK, I'll fix.